features/admin.api-resources.v1/components/api-resources-list.tsx

```diff
@@ -96,7 +96,10 @@ export const APIResourcesList: FunctionComponent<APIResourcesListProps> = (
     const [ loading, setLoading ] = useState<boolean>(false);
     const [ showDeleteConfirmationModal, setShowDeleteConfirmationModal ] = useState<boolean>(false);
     const [ deletingAPIResource, setDeletingAPIResource ] = useState<APIResourceInterface>(undefined);
-    const allowedScopes: string = useSelector((state: AppState) => state?.auth?.allowedScopes);
+
+    const hasApiResourcesReadPermissions: boolean = useRequiredScopes(featureConfig?.apiResources?.scopes?.read);
+    const hasApiResourcesUpdatePermissions: boolean = useRequiredScopes(featureConfig?.apiResources?.scopes?.update);
```

Comment:
The variable names convey a meaning that it holds whether api resources has CRUD permissions. 
Ideally these variables should convey that it holds whether the **user** has the relevant permissions.

To further clarify,

**Current version:**

Variable names suggest API resource permissions.

Example variable name: hasApiResourcesReadPermission

**Proposed version:**

Variable names should suggest user permissions.

Example variable name: userHasApiResourceReadPermission

Reply:
Seems like the current version is the convention we follow atm. Hence resolving.